core/src/main/java/com/graphhopper/routing/util/EncodingManager.java

```diff
@@ -42,6 +42,7 @@
 public class EncodingManager
 {
     public static final String CAR = "car";
+    public static final String CARSTOPOVER = "carstopover";
```

Comment:
same here: CAR_STOPOVER = "car_stopover" or do we need it now at all as the FastestWeighting is directly influenced?